### ISSUE TITLE
Use new sam user info end point

### ISSUE THREAD

**Issue by bradtaylor** [DataBiosphere/saturn-ui#504]
Once GAWB-3720 is done, use the new SAM api to get user status, rather than orchestration /me. This is needed for improved system stability.

Blocked on the backend ticket referred to above

**Comment by bradtaylor** [comment]
Duplicate of #500
